db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Product.destroy_all
+Location.destroy_all
+
+10.times do
+  FactoryGirl.create(:product)
+  FactoryGirl.create(:location)
```

Comment:
@ski-climb this is meant for development so we can work with realistic data. We probably won't seed production.